CRM/Odoosync/Hook/PostProcess/ContactSyncInformationUpdater.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * Updates contact sync information custom fields: sync_status, action_to_sync, action_date
+ */
+class CRM_Odoosync_Hook_PostProcess_ContactSyncInformationUpdater {
+
+  /**
+   * Id of sync_status custom field
+   *
+   * @var int
+   */
+  private $syncStatusFieldId;
+
+  /**
+   * Id of action_to_sync custom field
+   *
+   * @var int
+   */
+  private $actionToSyncFieldId;
+
+  /**
+   * Id of action_date custom field
+   *
+   * @var int
+   */
+  private $actionDateFieldId;
+
+  /**
+   * Current date time
+   *
+   * @var string
+   */
+  private $currentDateTime;
+
+  /**
+   * Value id of option awaiting_sync
+   *
+   * @var int
+   */
+  private $awaitingSyncOptionValueId;
+
+  /**
+   * Sets contact custom fields id: sync_status, action_to_sync, action_date
+   * Sets current data time
+   *
+   * CRM_Odoosync_Hook_PostProcess_ContactSyncInformationUpdater constructor.
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  public function __construct() {
+    $date = new DateTime();
+    $this->currentDateTime = $date->format('Y-m-d H:i:s');
```

Comment:
please add space after this, also you can change these two lines to one like this : 

```php
$this->currentDateTime =  (new DateTime())->format('Y-m-d H:i:s');
```